scripts/initdb_and_run.sh

```diff
@@ -4,6 +4,7 @@ if [ ! -e /app/.cache/mysql/db.done ]; then
     echo "Initializing DB..."
     PYTHONPATH=${PYTHONPATH}:/app/vendor/lib/python python /app/migrate_repo/manage.py version_control mysql://shipit:shipitpw@shipitdb/shipit migrate_repo
     PYTHONPATH=${PYTHONPATH}:/app/vendor/lib/python python /app/migrate_repo/manage.py upgrade mysql://shipit:shipitpw@shipitdb/shipit migrate_repo
+    bunzip2 -c /app/scripts/sample-data.sql.bz2 | mysql -h shipitdb -u shipit --password=shipitpw shipit
```

Comment:
If we decide to change something in the db schema, this may not work. How about something like
```bash
# the dump corresponds to db schema version 17
manage.py upgrade --version=17 ...
bunzip2 ... | mysql ...
# upgrade to the latest schema version
manage.py upgrade ...
```
I think this way it should work even if we update the schema.